src/scenes/level_scene.py

```diff
@@ -318,7 +318,16 @@ def load_level_content(self) -> None:
             # Game is loaded from a save (data)
             from_save = True
             gap_x, gap_y = (0, 0)
-            self.players.extend(loader.load_players(self.data))
+            if (self.map["width"] / TILE_SIZE) % 2 == 1:
```

Comment:
I believe int division could be used in this case (`//`), since we are always expecting map width to be a multiple of TILE_SIZE.

It would simplify the readability of the code.

Reply:
You are right. I'll change now.